README.md

```diff
@@ -70,7 +70,12 @@ const options = {
   logger                           // [optional] logger function to be called with errors
 };
 
-createTerminus(server, options);
+const terminus = new Terminus(server, options);
+
+// Returns the given server instance
+server = terminus.getHttpServer();
```

Comment:
can we just say `terminus.server`? I am thinking something around the lines of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get#get_Vs._defineProperty